api/ReadableStreamBYOBReader.json

```diff
@@ -331,12 +361,24 @@
             "edge": {
               "version_added": "89"
             },
-            "firefox": {
-              "version_added": false,
-              "notes": "See <a href='https://bugzil.la/1604037'>bug 1604037</a>."
-            },
+            "firefox": [
```

Comment:
Note this is implemented,  and the bug number is the same as all the others were fixed in. So I'm pretty certain this is an omission and I'm making it match the others.